packages/components/src/components/tabs/tabs.jsx

```diff
@@ -148,6 +148,7 @@ const Tabs = ({
                                     active_icon_color={active_icon_color}
                                     className={className}
                                     count={count}
+                                    children_count={valid_children.length}
```

Comment:
You don't need the prop `children_count`, you can set `tab_width` in this file, like on line 111.

Reply:
it is unset in this case

Reply:
Just give a condition `if (is_scrollable && valid_children.length > 3)`.

Reply:
if you mean to set a condition for class 'dc-tabs__item--small' then I still need to pass children_count

Reply:
@carol-binary so, the thing I'm adding children-count is that the width of the tabs is different when it's mobile and there are more than 3 tabs, that's why I need a new class, and on dp2p we need tab_width unset anyway

Reply:
If you need to give it a class then why not just give it in DP2P?

Reply:
sure